ott/geometry/costs.py

```diff
@@ -48,6 +80,10 @@ def pairwise(self, x: jnp.ndarray, y: jnp.ndarray) -> float:
   def barycenter(self, weights: jnp.ndarray, xs: jnp.ndarray) -> float:
     pass
 
+  @classmethod
+  def padding_vector(cls, dim: int) -> jnp.ndarray:
```

Comment:
maybe rename this to avoid having both the vector and the function be called the same thing, e.g. `padding_for_cost` or something of this sort?